phpBB/phpbb/install/converter/controller/helper.php

```diff
@@ -0,0 +1,208 @@
+<?php
```

Comment:
It has additional functions for me to load and save certain details regarding the converter. It extends the install helper.